<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QWidget" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>442</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Sign Up</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="Email">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Username">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Re-write Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FirstName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LastName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="Login">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Already have an account? Log in</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>400</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QLabel" name="alert">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Instructions for Signing Up:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Email: Must be a valid email address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username: Can't be more than 50 characters</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>First and Last Names: These two may be left empty if you wish but not advisable</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
